### PR TITLE
Enable Docker Hub to build image automatically on push

### PR DESCRIPTION
**Pull request by rtau**
By merging the docker-pinry content, it would allow Docker Hub to monitor the repository for changes and trigger an automated build.

### CODE REVIEW

File: docker-contents/README.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+Getting docker-pinry
+---------------------
+
+Running this will get the latest version of both
+docker-pinry and pinry itself::
+
+  git clone https://github.com/pinry/docker-pinry
+  cd docker-pinry
+  ./bootstrap.sh
+
```
Comment (winkidney) [inline docker-contents/README.rst]: Please also help to adapt the document to latest version of our building method.
1. The script path
2. How to config the docker and config file.

Thanks.

File: docker-contents/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```
Comment (winkidney) [inline docker-contents/bootstrap.sh]: Does this script still work with current directory structure?

Comment (rtau) [reply docker-contents/bootstrap.sh]: Haha, I was not even aware of those scripts as I was too concentracted on the autobuild on Docker Hub. They have now been updated to work after the merge. The docker-contents/README.rst had been updated as well.

File: docker-contents/README.rst (hunk 1 above)
Comment (rtau) [reply docker-contents/README.rst]: I've updated the paths in the README.rst

**Comment by winkidney** [comment]
Sorry for my slow response for your awesome jobs! 
I will make some tests on my own instance and them give you a final review once I have time !

**Comment by winkidney** [comment]
Merged, thanks for your contribution! And may be we need a account to setup auto build.

**Pull request merged**